extensions/adobe/experience/journey/orchestration/journeyStepEventActionExecutionFieldsMixin.Json

```diff
@@ -0,0 +1,228 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/journey/orchestration/journeyStepEventCommonFields",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Journey Orchestration Step Data Fetch Fields",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": [
+    "https://ns.adobe.com/xdm/context/experienceevent"
+  ],
+  "description": "This Mixin contains the common Jounney Metadata information ",
+  "definitions": {
+    "journeyStepEventCommonFields": {
+      "properties": {
+        "https://ns.adobe.com/experience/journey/orchestration": {
+          "title": "Journey Orchestration for creating user journey.",
+          "type": "object",
+          "properties": {
+            "entrance":{
+              "title":"entrance",
+              "description":"Indicate if the user has entered the journey. If not present, assuming that the value is false",
+              "type":"boolean",
+              "isRequired":false,
+              "default":true,
+              "meta:xdmType":"boolean"
+            },
+            "reentrance":{
+              "title":"reentrance",
+              "description":"Indicate if the user has reentered the journey with the same instance. If not present, assuming that the value is false",
```

Comment:
The description implies that this defaults to false, but the `default` property is set to `true`.